.composer-require-checker.json

```diff
@@ -0,0 +1,16 @@
+{
+    "symbol-whitelist" : [
+        "HHVM_VERSION_ID", "T_HH_ERROR",
+        "PHPUnit\\Framework\\TestCase", "Prophecy\\Argument",
+        "GeckoPackages\\PHPUnit\\Constraints\\SameStringsConstraint",
```

Comment:
With `gecko-packages/gecko-php-unit` in the require (https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.5/composer.json#L21) part if `composer.json` this should no longer be needed.
Just a note to myself to check on this.